components/discord_bot/actions/create-guild-channel/create-guild-channel.mjs

```diff
@@ -0,0 +1,159 @@
+import common from "../common.mjs";
```

Comment:
Since this component doesn't use `channelId`, we should include the common.mjs file in the discord_bot directory.
`import common from "../../common.mjs"`